docs/source/publish_and_share_kedro_viz_on_aws.md

```diff
@@ -0,0 +1,118 @@
+
+![](./images/kedro_viz_share_credentials2.png)
+
+For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
+
+Step 6: Add the IAM user to the user group (this is only possible if the group has been created).
+
+![](./images/kedro_viz_share_credentials3.png)
+
+Step 7: Select the user, then select `Create access key`. Follow the steps and create your keys.
+
+![](./images/kedro_viz_share_credentials4.png)
+
+### Permissions and access control
+
+All permissions and access control are controlled by AWS. It's up to you, the user, if you want to allow anyone to see your project or limit access to certain IP addresses, users, or groups.
+
+You can control who can view your visualisation using [bucket and user policies](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-iam-policies.html) or [access control lists](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acls.html). See the official AWS documentation for more information.
+
+## Set credentials
+
+Step 8: Once that's completed, you'll need to set your AWS credentials as environment variables in your terminal window, as shown below:
+
+```bash
+export AWS_ACCESS_KEY_ID="your_access_key_id"
+export AWS_SECRET_ACCESS_KEY="your_secret_access_key"
+```
+
+For more information, see the official AWS documentation about [how to work with credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
+
+
+### Billing
+
+You pay for storing objects in your S3 buckets. The amount you pay depends on your objects’ size, how long you stored the object during the month, and the storage class.
+
+See the official [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information.
+
+## Publish and share the project
+
+Step 9: Once your Cloud storage is configured and the credentials are set, you are now ready to publish and share your Kedro-Viz project. Start Kedro-Viz by running the following command in your terminal:
+
+```bash
+kedro viz run
+```
+
+Step 10: Click the **Publish and share** icon in the lower-left of the application. You will see a modal dialog to select your relevant AWS Bucket Region and enter your Bucket Name.
+
+```{note}
+From Kedro-Viz version 8.0.0, you will see a modal dialog to select your hosting platform, input your bucket name and endpoint link. The endpoint link can be found under **S3 bucket -> properties -> Static website hosting -> Bucket website endpoint**.
+```
+
+Step 11: Once those details are complete, click **Publish**. A hosted, shareable URL will be returned to you after the process completes.
+
+Here is an example of the flow (TODO - Need to add flows specific to cloud provider):
+
+![](./images/kedro-publish-share.gif)
+
+```{note}
+We will be updating the user flow doc for v8.0.0 soon...
```

Comment:
I wouldn't add this, it seems a bit sloppy and the user can't do anything with this information.